### PR TITLE
Updates the workshop to use Parachain Template

### PR DESCRIPTION
**Pull request by JoshOrndorff**
This PR updates the workshop to latest cumulus (and thus also latest substrate and polkadot). It also ditches the cumulus-test-parachain-collator in favor of the Substrate Parachain Template. And it makes the docker images actually work.

### CODE REVIEW

File: 3-parachains/2-register.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ The transaction can be made from `Apps > Sudo > Registrar > registerPara` with t
 
 id: 0
```
Comment (wischli) [inline 3-parachains/2-register.md:9]: Shouldn't this match the parachain id which we've set in 3.1, i.e.
```suggestion
id: 200
```

File: 3-parachains/1-launch.md [hunk 2]
```diff
@@ -1,39 +1,42 @@
...
 ## Start the Collator Node
 We can now start the collator node with the following command. Notice that we're supplying the same relay chain spec we used when launching relay chain nodes.
 
 ```bash
 test-collator \
-  --base-path collator-100 \
+  --base-path /tmp/para-200-collator-1 \
   --ws-port 9977 \
   --port 30336 \
   --parachain-id 100 \
```
Comment (wischli) [inline 3-parachains/1-launch.md:31]:
```suggestion
  --parachain-id 200 \
```

**Comment by JoshOrndorff** [comment]
I think this is good enough to merge and then fix up details in individual PRs.

**Comment by JoshOrndorff** [comment]
cc @tolak This solves #5

**Pull request merged**